frontend/src/Pages/OpenContent.tsx

```diff
@@ -41,7 +54,7 @@ export default function OpenContent() {
                         className="button border border-primary bg-transparent text-body-text"
                         onClick={() =>
                             navigate(
-                                `/open-content-management/${activeTab.value}`
+                                `/open-content-management/${String(activeTab.value).toLowerCase()}`
```

Comment:
If the admin is on the student view of the favorites and they click return to admin view, an error will be thrown since there is no such thing as a favorites page for an admin. Here we will probably have to check their location and what role they are trying to switch to, and handle the case that they are on the favorites page trying to switch to admin view and just direct them to the libraries tab.
<img width="1512" alt="Screenshot 2024-11-22 at 10 53 08 AM" src="https://github.com/user-attachments/assets/c89eeae5-ca12-4246-9886-ed2ee172f8e5">

<img width="1512" alt="Screenshot 2024-11-22 at 10 49 56 AM" src="https://github.com/user-attachments/assets/9200cbf4-7fe7-4111-a9d6-2aef352051fb">